### ISSUE TITLE
Signups on etherpad are hard to navigate

### ISSUE THREAD

**Issue by brownsarahm** [carpentries/conversations#39]
## 4. Visual noise and Task Switching (Etherpads)

I found the Etherpad useful during the lesson exercises, when participants were populating content into the page shared between the group. However, the instructor checkout process requires signing up for a community discussion and a teaching demo using the pads at https://pad.carpentries.org/community-discussions and https://pad.carpentries.org/teaching-demos. 

The information density of these two pages can be stressful when just five minutes is allocated for signing up within the training schedule. Being presented with this page (with no indexes or hyperlinks to find specific sections within it!) is a high amount of visual noise and it is difficult to navigate, especially when attention is also split with the Zoom call. Philipp handled this really well and clarified that I could come back to these later, but this could be emphasised within the teaching material, or perhaps a more streamlined way could be found to track sign ups to these sessions.

This could also be due to difficulties task switching: changing from a lecture-based learning approach to having to interpret large amounts of data.

_Originally posted by @sjmf in https://github.com/carpentries/instructor-training/issues/1395#issuecomment-1163278573_

I also received this feedback separately from another trainee.

**Comment by acrall** [comment]
@klbarnes20 Can you respond to this issue with relevant information from the checkout project?

**Comment by klbarnes20** [comment]
Thanks @acrall. Yes, as part of the recommendations to come out of the checkout project, we are committed to exploring alternatives to using Etherpad for Community Discussion and Teaching Demo sign ups. We will be working on an implementation plan in the coming days. However, the timeline on this is uncertain as there are logistical and technical challenges as we consider alternatives.

**Comment by kaijagahm** [comment]
Just discovered this through the original post in #1395 and wanted to say that I also find the Etherpads hard to navigate and the signups to be overwhelming! It would be nice if there were a single signup link with automatic calendar integration instead.

**Comment by brownsarahm** [comment]
I believe alternative tools are still in the pilot phase, but @maneesha may be able to give an update.

**Comment by maneesha** [comment]
Yes, we are considering alternatives to the etherpads as we all recognize they are difficult to use.  We are piloting this for Welcome Sessions for 2025 Q4 and will explore other options over 2026.  See more info on the etherpad:  https://pad.carpentries.org/welcome-sessions-2025
